### PR TITLE
Issue#13 Add Unit Tests

### PR DESCRIPTION
**Pull request by tnorimat**
https://github.com/webauthn4j/keycloak-webauthn-authenticator/issues/13

### CODE REVIEW

File: webauthn4j-ejb/pom.xml [hunk 1]
```diff
@@ -43,6 +71,24 @@
             <groupId>com.webauthn4j</groupId>
             <artifactId>webauthn4j-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```
Comment (ynojima) [inline webauthn4j-ejb/pom.xml]: Keycloak is using junit 4.12. 
https://github.com/keycloak/keycloak/blob/master/pom.xml#L132

It is better to align it for aiming to merge into keycloak.

Comment (tnorimat) [reply webauthn4j-ejb/pom.xml]: I see. I will rewrite test codes by using junit 4.12.

File: webauthn4j-ejb/src/test/java/org/keycloak/authenticator/RegisterAuthenticatorTest.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                public Object answer(InvocationOnMock invocation) {
+                    Object[] args = invocation.getArguments();
+                    Object mock = invocation.getMock();
+                    Response response = (Response) args[0];
+                    System.out.println("WWWWWWWWWW called with arguments: " + response.getClass().getCanonicalName());
+                    return null;
+                }
+            }
+        ).when(context).challenge(any(Response.class));
+        */
```
Comment (ynojima) [inline webauthn4j-ejb/src/test/java/org/keycloak/authenticator/RegisterAuthenticatorTest.java]: Is this comment needed to be kept?

Comment (tnorimat) [reply webauthn4j-ejb/src/test/java/org/keycloak/authenticator/RegisterAuthenticatorTest.java]: Sorry. I've forgot to get rid of them.

**Review by ynojima — APPROVED** [review]

**Pull request merged**